commands/delete.go

```diff
@@ -0,0 +1,118 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/github/hub/git"
+	"github.com/github/hub/github"
+	"github.com/github/hub/utils"
+)
+
+var cmdDelete = &Command{
+	Run:   delete,
+	Usage: "delete [[<ORGANIZATION>/]<NAME>]",
```

Comment:
Instead of making `NAME` optional, how about requiring the argument? That would force the user to type the name of the repository to delete even if it's the current repository.

Reply:
Agreed, delete the current working repo is a little bit dangerous anyway. :-)